frontend/.vscode/settings.json

```diff
@@ -22,6 +22,7 @@
     "NotificationsNetworkTable",
     "NotificationsOverview",
     "NotificationsTableEmpty",
+    "SummaryDetails",
```

Comment:
we should never add wrong ScopeNames -> `DashboardSummaryDetails` 

would be nice to rename the component then

Reply:
definitely, thanks for the catch :pray: